next.clayui.com/content/docs/css/components/cards.md

```diff
@@ -0,0 +1,1955 @@
+</div>
+
+```html
+<div class="card card-interactive card-interactive-primary card-type-template template-card active" tabindex="0">
+    <div class="aspect-ratio card-item-first">
+        <div class="aspect-ratio-item aspect-ratio-item-center-middle aspect-ratio-item-flush">
+            <img src="/images/portlet.svg">
+        </div>
+    </div>
+    <div class="card-body">
+        <h3 class="card-title">Widget Page</h3>
+        <div class="card-text">Build a page by adding widgets and content.</div>
+    </div>
+</div>
+```
+
+## Empty
+
+By default, when adding `image-card` class and inside the element that contains `image-card`, exists a `aspect-ratio` class. A transparent background will be setted.
+
+<div class="sheet-example">
+    <div class="col-md-4">
+        <div class="card card-type-asset image-card">
+            <div class="aspect-ratio card-item-first">
+                <span class="sticker sticker-bottom-left sticker-info">PNG</span>
+            </div>
+            <div class="card-body">
+                <div class="card-row">
+                    <div class="autofit-col autofit-col-expand">
+                        <section class="autofit-section">
+                            <h3 class="card-title" title="empty-background.png">
+                                <span class="text-truncate-inline">
+                                    <a class="text-truncate" href="#1">empty-background.png</a>
+                                </span>
+                            </h3>
+                            <p class="card-subtitle" title="Author Action">
+                                <span class="text-truncate-inline">
+                                    <span class="text-truncate">Author Action</span>
+                                </span>
+                            </p>
+                        </section>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+
+```html
+<div class="card card-type-asset image-card">
+    <div class="aspect-ratio card-item-first">
+        <span class="sticker sticker-bottom-left sticker-info">PNG</span>
+    </div>
+    <div class="card-body">
+        <div class="card-row">
+            <div class="autofit-col autofit-col-expand">
+                <section class="autofit-section">
+                    <h3 class="card-title" title="empty-background.png">
+                        <span class="text-truncate-inline">
+                            <a class="text-truncate" href="#1">empty-background.png</a>
+                        </span>
+                    </h3>
+                    <p class="card-subtitle" title="Author Action">
+                        <span class="text-truncate-inline">
+                            <span class="text-truncate">Author Action</span>
+                        </span>
+                    </p>
+                </section>
+            </div>
+        </div>
+    </div>
+</div>
+```
+
+# Helpers
+
+## Checkbox
```

Comment:
I would not consider the checkbox a Helpers, it should be a variant. I would consider a helper a class util that helps to do something, align, spacing...